packages/node_modules/@ciscospark/redux-module-conversation/src/actions.test.js

```diff
@@ -0,0 +1,85 @@
+/* eslint-disable max-nested-callbacks */
+import configureMockStore from 'redux-mock-store';
+import thunk from 'redux-thunk';
+
+const middlewares = [thunk];
+const mockStore = configureMockStore(middlewares);
+
+const mockSpark = {
+  conversation: {
+    acknowledge: jest.fn(() => Promise.resolve()),
+    create: jest.fn(() => Promise.resolve({id: `created-conversation`})),
+    delete: jest.fn(() => Promise.resolve()),
+    get: jest.fn(() => Promise.resolve({id: `got-conversation`})),
+    listActivities: jest.fn(() => Promise.resolve([{mockActivity: `mockActivity`}]))
+  }
+};
+
+import * as actions from './actions';
+
+describe(`redux-module-conversation actions`, () => {
+  it(`has exported actions`, () => {
+    expect(actions.acknowledgeActivityOnServer).toBeDefined();
+    expect(actions.createConversation).toBeDefined();
+    expect(actions.deleteActivity).toBeDefined();
+    expect(actions.getConversation).toBeDefined();
+    expect(actions.loadMissingActivities).toBeDefined();
+    expect(actions.loadPreviousMessages).toBeDefined();
+  });
+
+  it(`can acknowledge an activity`, () => {
+    const store = mockStore({});
+    const activity = {id: `activity-to-acknowledge`};
+    return store.dispatch(actions.acknowledgeActivityOnServer({toJS: jest.fn()}, activity, mockSpark))
```

Comment:
Maybe create a constructor for the Map instead of using `{toJS: jest.fn()}`? Took me way too long to figure out what that was.